arangod/IResearch/IResearchLinkCoordinator.cpp

```diff
@@ -83,6 +83,77 @@ IResearchLinkCoordinator::IResearchLinkCoordinator(
   _sparse = true;   // always sparse
 }
 
+Result IResearchLinkCoordinator::init(
+    velocypack::Slice definition,
+    IResearchLink::InitCallback const& initCallback) {
+  auto r = IResearchLink::init(definition, initCallback);
+  auto& metric = IResearchLink::collection()
+                     .vocbase()
+                     .server()
+                     .getFeature<metrics::ClusterMetricsFeature>();
+  std::function batchToCoordinator =
+      [](metrics::ClusterMetricsFeature::CoordinatorMetrics& metrics,
+         std::string_view name, velocypack::Slice labels,
+         velocypack::Slice value) {
+        auto& cell = std::get<uint64_t>(
+            metrics[metrics::ClusterMetricsFeature::MetricKey{
+                name, labels.stringView()}]);
+        cell += value.getUIntUnchecked();
+      };
+  metrics::ClusterMetricsFeature::ToPrometheus batchToPrometheus{
+      [](std::string& result, std::string_view name) {
+        result.append("# HELP ").append(name);
+        result.push_back(' ');
+        result.append("NO HELP");  // TODO(MBkkt) Should be help string
```

Comment:
Will this be addressed in a follow-up PR?

Reply:
Yes, of course

Its about pr with one source of true for metrics